front/package.json

```diff
@@ -49,7 +49,7 @@
   "dependencies": {
     "@apollo/client": "^3.5.6",
     "@babel/runtime": "7.x",
-    "@citizenlab/cl2-component-library": "^0.9.68",
+    "@citizenlab/cl2-component-library": "^0.9.70",
```

Comment:
Nice! Could you please also implement the new icon in the content builder white space component?

Reply:
@IvaKop This has been updated